scripts/create_db.sql

```diff
@@ -0,0 +1,61 @@
+select 'create database ita'
+where not exists(select from pg_database where datname = 'ita');
+\gexec
+
+\c ita
+
+create schema if not exists converter;
+
+do $$
+    begin
+        if not exists(select 1 from pg_extension where extname = 'uuid-ossp') then
+            create extension "uuid-ossp";
+        end if;
+    end
+$$;
+
+do $$
+    begin
+        if not exists(select 1 from pg_type where typname = 'file_format') then
+            create type file_format as enum ('jpeg', 'png');
+        end if;
+    end
+$$;
+
+do $$
+    begin
+        if not exists(select 1 from pg_type where typname = 'status') then
+            create type status as enum ('queued', 'processed', 'failed', 'done');
+        end if;
+    end
+$$;
+
+create table if not exists converter.users (
+    id uuid default uuid_generate_v1() primary key,
+    email varchar(50) unique not null,
+    password varchar(50) not null
+);
+
+create table if not exists converter.images (
+    id uuid default uuid_generate_v1() primary key,
+    name varchar(80) not null,
+    format file_format not null,
+    location text not null
+);
```

Comment:
user and image is also created somewhen, it's a good practice to store this info too